tests/e2e/specs/modules/analytics/setup-with-account-no-tag.test.js

```diff
@@ -168,4 +171,21 @@ describe( 'setting up the Analytics module with an existing account and no exist
 		await expect( page ).toMatchElement( '.googlesitekit-analytics__select-property .mdc-select__selected-text', { text: '' } );
 		await expect( page ).toMatchElement( '.googlesitekit-analytics__select-profile .mdc-select__selected-text', { text: '' } );
 	} );
+
+	describe( 'Homepage AMP', () => {
+		it( 'validates for logged-in users', async () => {
+			await setupAnalytics();
+			await activateAmpAndSetMode( 'standard' );
+			await Promise.all( [
+				page.goto( createURL( '/' ), { waitUntil: 'load' } ),
+				page.waitForSelector( '#amp-admin-bar-item-status-icon' ),
+			] );
+			await expect( page ).toMatchElement( '#amp-admin-bar-item-status-icon', { text: '✅' } );
```

Comment:
We should extract this to a custom matcher as well I think. These could actually use the same matcher, but it would need to accept something like `{ loggedIn: true }` as a second argument to determine the method used for validation. I don't think it should be responsible for activating or configuring the AMP plugin however. By default I think it should use the official validator, testing the html visitors see as it does now.